recipes/uppasd/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set version = "v6.0.2" %}
+
+package:
+  name: uppasd-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/UppASD/UppASD/releases/download/{{ version }}/{{ version }}.tar.gz
+  sha256: f483bb6e0a5c75820e4a3d38894c40557548777160eabd3c152c8c356e6ca8ed
+  patches:
+    - patches/0001-Change-name-of-binary-from-sd-to-uppasd.patch
+    - patches/0002-Remove-copying-of-static-library-to-outside-build-directory.patch
+    - patches/0003-Remove-compiler-optimization-for-host-architecture.patch
+    - patches/0004-Remove-lgfortran-which-is-not-supported-by-flang.patch  # [win]
+    - patches/0005-Replace-external-definitions-with-use-omp_lib.patch  # [win]
+    - patches/0006-Add-size-argument-to-transfer-calls.patch  # [win]
+    - patches/0007-Fix-print-formatting-syntax-errors.patch  # [win]
+
+build:
+  number: 0
+
+test:
+  source_files:
+    - examples/SimpleSystems/bccFe
+  commands:
+    - cd examples/SimpleSystems/bccFe && uppasd  # [not win and build_platform == target_platform]
```

even if/once we add cross-compiled platforms on the feedstock, we don't need this. Some architectures can be emulated through QEMU (aarch/ppc), and the others are skipped through the config in `conda-forge.yml`, which should then be `test: native_and_emulated`.
```suggestion
    - cd examples/SimpleSystems/bccFe && uppasd  # [unix]
```